e2echart/e2e-chart-template.html

```diff
@@ -115,6 +152,13 @@ <h5 class="modal-title">Resource</h5>
     }
 
     function isPod(eventInterval) {
+        const [isLeaderFoundOrLost, notUsed] = isEtcdLeaderFoundOrLost(eventInterval)
+        if (isLeaderFoundOrLost) {
+            return false
+        }
+        if (eventInterval.locator.includes("etcd-member/")) {
+            return false
+        }
```

Comment:
This callout to isEtcdLeaderFoundOrLost looks still required until we fully port the file. This isPod section in particular looks susceptible to showing things because it doesn't check tempSource.